aws-predictions/src/test/java/com/amplifyframework/predictions/aws/http/LivenessWebSocketTest.kt

```diff
@@ -317,6 +325,57 @@ internal class LivenessWebSocketTest {
         assertEquals(livenessWebSocket.getUserAgent(), "$baseline $additional")
     }
 
+    @Test
+    fun `web socket detects clock skew from server response`() {
+        val livenessWebSocket = createLivenessWebSocket()
+        mockkConstructor(WebSocket::class)
+        val socket: WebSocket = mockk()
+        livenessWebSocket.webSocket = socket
+        val sdf = SimpleDateFormat(livenessWebSocket.datePattern, Locale.getDefault())
```

Comment:
The SimpleDateFormat is currently using Locale.getDefault(), which might introduce inconsistencies with date parsing if the default locale does not match the expected server's date header format, potentially leading to false negatives in tests.

Suggestion:

```kotlin
val sdf = SimpleDateFormat(livenessWebSocket.datePattern, Locale.US)
```
Using Locale.US is a common practice for parsing HTTP dates, as per RFC 1123.

Reply:
`Locale.getDefault()` does throw an error, thanks @ankpshah for the callout -- fixed in most recent commit